test/test_container.py

```diff
@@ -94,3 +94,32 @@ def test_container_with_progress(tmp_path, build_fake_container, progress, needl
     ], text=True)
     assert needle in output
     assert forbidden not in output
+
+
+def test_container_cross_build_riscv64(tmp_path, build_container):
+    # XXX: we should encode this in images proper
+    # note that we need the "fedora-toolbox" container as the regular one
+    # does not contain python3
+    images_experimental_env = "IMAGES_EXPERIMENTAL=buildroot=fedora.riscv.rocks:3000/davidlt/fedora-toolbox:41"
```

Comment:
This will need to use ghcr.io/mvo5/fedora-buildroot:41

which is just:
```Containerfile
FROM fedora.riscv.rocks:3000/davidlt/fedora-toolbox:41
RUN dnf install -y selinux-policy-targeted policycoreutils dosfstools
```
#136 opened Feb 18, 2025 by [mvo5](https://github.com/osbuild/image-builder-cli/issues?q=is%3Apr+is%3Aopen+author%3Amvo5)

Reply:
Note that this is an intermediate solution, once the upstream `fedora-toolbox` container installs policycoreutils this container is no longer needed and we can use the upstream one.